resources/views/livewire/project/shared/get-logs.blade.php

```diff
@@ -48,8 +48,8 @@
         </div>
         <form wire:submit='getLogs(true)' class="flex gap-2 items-end">
             <div class="w-96">
-                <x-forms.input label="Only Show Number of Lines" placeholder="1000" type="number" required
-                    id="numberOfLines"></x-forms.input>
+                <x-forms.input label="Only Show Number of Lines" placeholder="100" type="number" required
```

Comment:
I figured out what's causing this. 
- Livewire tries to set `numberOfLines` to null because the input is empty
- The configured type in GetLogs is only `int` so it throws a TypeError
- Livewire hides the error and `unset()`s the property instead: https://github.com/livewire/livewire/blob/0be2dbff88639520d870127c2e02ee907e260c7a/src/Mechanisms/HandleComponents/HandleComponents.php#L430

Reply:
Thanks for the investigation @KaelWD! 

I will set it to be nullable and if it null, set it to the default value.